app/gateway/2.8.x/install-and-run/helm-quickstart-enterprise.md

```diff
@@ -0,0 +1,323 @@
+---
+title: How to Install Kong Gateway with Helm
+toc: false
+---
+
+This guide shows you how to deploy {{site.base_gateway}} with Helm on your local Kubernetes environment. These steps are known to work on [Docker Desktop Kubernetes](https://docs.docker.com/desktop/kubernetes/) and [Kind](https://kind.sigs.k8s.io/).
+Once deployed, Kong will be locally accessible at `https://kong.7f000001.nip.io`. [nip.io](https://nip.io) is used to automatically resolve this domain to localhost. 
+
+The {{site.base_gateway}} software is governed by the
+[Kong Software License Agreement](https://konghq.com/kongsoftwarelicense/).
+{{site.ce_product_name}} is licensed under an
+[Apache 2.0 license](https://github.com/Kong/kong/blob/master/LICENSE).
+
+
+{% navtabs %}
+{% navtab Docker Desktop Kubernetes %}
+## Install Kong Gateway on Docker Desktop
+## Prerequisites 
+
+- [`Helm 3`](https://helm.sh/)
+- [`kubectl`](https://kubernetes.io/docs/tasks/tools/) v1.19 or later
+- A `license.json` enterprise license file from Kong
+- [Docker Desktop](https://docs.docker.com/desktop/#download-and-install) with Kubernetes enabled. 
+
+
+## Install Dependencies
+
+Docker Desktop does not enable Kubernetes functionality automatically. To enable Docker Desktop Kubernetes, open the Docker application on your local machine, click the **Settings Cog** in the top-right corner of the application to open **Preferences**. From the **Preferences** menu, click the Kubernetes option, and **Enable Kubernetes**.
+
+Once Docker Desktop Kubernetes is ready, let's install dependencies: 
+
+1. Add the Jetstack Cert Manager Helm repository:
+
+        helm repo add jetstack https://charts.jetstack.io ; helm repo update
+
+2. Install Cert Manager:
+        
+        helm upgrade --install cert-manager jetstack/cert-manager \
+            --set installCRDs=true --namespace cert-manager --create-namespace
+
+## Configure Kong Gateway
+
+Configuring Kong Gateway requires a Namespace and configuration secrets. Our secrets contain Kong's enterprise license, admin password, session configurations, and Postgres connection details. If you do not have a `license.json` file, please contact your account manager.
+
+2. Create Kong Namespace for {{site.base_gateway}}:
+
+        kubectl create namespace kong --dry-run=client -oyaml | kubectl apply -f -
```

Comment:
we can simplify this down to the `k create ns kong` minimal command syntax. My primary reason for this syntax was to avoid exiting with an error if it was run twice by someone.